scripts/pre_push_hook.py

```diff
@@ -295,6 +295,19 @@ def _install_hook():
         print 'Copied file to .git/hooks directory'
 
 
+def run_frontend_tests_status(files_to_lint):
+    """Checks if frontend tests needs to be run."""
+
+    js_files_to_check = [
+        filename for filename in files_to_lint if
+        filename.endswith('.js')]
+
+    if js_files_to_check == []:
```

Comment:
Replace all this by: `return bool(js_files_to_check)`.

Reply:
Oh Thanks @seanlip, nice idea. :)

Reply:
Done